Add email announcing SR2021 (#65)
srobo/team-emails: SR2021/2020-10-29-sr2021-announcement.md

---
to: Potential team leaders
subject: Announcing Student Robotics 2021
---

Hello there!

> I'm pleased to announce some details for Student Robotics 2021.

Reviewer: ```suggestion
We're pleased to announce some details for Student Robotics 2021.
```

Will this email be sent from a personal account or from teams@ (or similar)? If from a shared mailbox and there's no no signature explicitly naming a person feel we should go for 'We' rather than 'I'

## Competition

> Unfortunately, we are unable to run a physical competition in a safe way. Instead, we will be running a virtual competition in a simulated environment. The competition year will be very similar to what you're used to. In a few weeks, we'll kick off the competition with a kickstart, which will culminate in a final competition 1st May.

Reviewer: Feel like we should highlight previous virtual competition. Reworded last sentences as well but take the changes or leave them I don't feel super strongly

```suggestion
Unfortunately, we are unable to find a way run a physical competition in a COVID safe way. Instead, following on the success of our [previous virtual competition](https://studentrobotics.org/news/2020-07-25-post-competition/) we will be running the competition in a simulated environment. The competition's time scale will be very similar to what you're used to, with just a few short weeks until we'll kick off the competition with our Kickstart event. Teams will then have roughly 7 months before the final competition 1st May.
```

However, on top of this we will be running monthly league sessions. These league sessions will be similar to those run for the virtual competition, and much like the regular competition league will seed your position in the final knockout.

## Kickstart

> Due to current restrictions, kickstart will be streamed live on **21st November 2020**. Here we'll announce the game for this year, go into further details on how the competition year will work, and of course answer any questions you have.

Reviewer 2: I think the reasons here are obvious enough to just skip to the content.
```suggestion
Kickstart will be streamed live on **21st November 2020**. Here we'll announce the game for this year, go into further details on how the competition year will work, and of course answer any questions you have.
```

> Visit our website for more information.

Reviewer: If https://github.com/srobo/website/pull/231 lands before this email is sent we should add a link.
